Added documentation of Blockly integration, with all new tutorials.

fuglaro opened the pull request:
Depends on this PR https://github.com/TinyCircuits/TinyCircuits-Thumby-Code-Editor/pull/66 from the TinyCircuits-Thumby-Code-Editor.

fuglaro committed:
Added documentation of Blockly integration, with all new tutorials.

Lennevia commented:
Sorry to leave you hanging! I've been keeping tabs on updates over on the code editor PR before I sit down with this, but I should've let you known explicitly that that's why there's a lag :)

fuglaro commented:
Thank you for checking in! No worries at all! Please do take your time,
there is zero rush. I can only imagine how many "Tiny" things need getting
done!

On Mon, 16 Jan 2023, 19:26 Laveréna Wienclaw, ***@***.***>
wrote:

> Sorry to leave you hanging! I've been keeping tabs on updates over on the
> code editor PR before I sit down with this, but I should've let you known
> explicitly that that's why there's a lag :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TinyCircuits/Thumby-Website/pull/10#issuecomment-1384468224>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLICBWEXGAE5O2DKRMFKRDWSWOFVANCNFSM6AAAAAATZKW4F4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

fuglaro committed:
Blockly: Exitor tabs and examples now have the .blocks extension instead of .py

Now matches https://github.com/TinyCircuits/TinyCircuits-Thumby-Code-Editor/pull/66/commits/4415e3e0dede5a8e310485ba5b17f263bce10e65

Lennevia commented:
In line with the code editor PR [#66 ](https://github.com/TinyCircuits/TinyCircuits-Thumby-Code-Editor/pull/66), I'll merge this in and continue reviewing for a day or so before building the site and making everything ✨LIVE✨. I'll be passing through for typos and any other formatting things I feel picky about. 
I did catch an error with a link that I'll fix since I know where it is, and I think I may add some 'Jump to' links for some of the longer tutorials if I decide the table of contents links aren't sufficient on my next read through. 

All those little disclaimers on possible nitpicking to say: you did a great job and I don't think there are any large structural changes I'll make. The amount of photos and steps taken with care sets a good standard and precedent for future documentation that should cater to beginners. I'm excited to see people use Blockly with Thumby!
